addons/l10n_za/data/account_tax_report_data.xml

```diff
@@ -25,7 +25,7 @@
         <field name="name">[19] Total B: TOTAL INPUT TAX (14 + 14A + 15 + 16 + 17 + 18)</field>
         <field name="report_id" ref="tax_report"/>
         <field name="sequence" eval="2"/>
-        <field name="formula">VAT14 + VAT14A + VAT15 + VAT16 + VAT17 + VAT18</field>
+        <field name="formula">VAT14 + VAT14A + VAT15 + VAT15A + VAT16 + VAT17 + VAT18</field>
```

Comment:
This is not in sync with the name anymore

Reply:
Please also use https://github.com/odoo/enterprise/commit/5d7ed69e423d6e6814a5a185279afd3b634c531b when applicable

Reply:
Interesting ! Nevertheless, I will keep this formula like this since it's the way the officials explain it.